Makefile

```diff
@@ -5,9 +5,7 @@ PYTEST=pytest
 GIT=git
 TOX=tox
 ICONV=iconv
-FLAKE8=flake8
```

Comment:
should we remove flake8 from make file?

Reply:
The pre-commit already has it, and we also run the pre-commit on the pipeline.

Unless... You use it? :eyes:

Reply:
makefile is for local use as far I can tell

Reply:
pre-commit as well:

`pre-commit run flake8 --all-files`

My point is just to not have multiple ways to do the same thing.